### DISCUSSION TITLE
Any placeholders on the line does not work

### DISCUSSION THREAD

**Discussion by aldrodav** [jgraph/drawio#4643] [General]
Hi
i have v24.7.8
and i try to use placeholders in diagram but on the line any placeholders does not work as i expected to add detail in my physical network diagram
![image](https://github.com/user-attachments/assets/60b6af9f-6142-4590-a1af-97558a811344)

i wil be glad and says many thanks you if we can fixup tish bug

**Comment by davidjgraph** [comment]
If you add labels to a connector, they are child shapes of the edge. Only the default (select and type) label of an edge is the label of the actual edge.

**Reply by aldrodav** [reply]
@davidjgraph thank you for reply, bat i confused with understanding edge concept int this article
i tryed to add property ifindex42 with value Fo1/1/41 and use this as placeholder of the link lable, but its dosent work also
however this property as placeholder in lable of the switch shape work fine
![image](https://github.com/user-attachments/assets/7d5016ac-2bff-46e6-b6c3-f53e5c6b36ac)

**Reply by elkram** [reply]
I have a similar question, and also don't understand your answer.
We'd like to have data driven connector labels and that doesn't seem to be supported.
I don't understand how being the child of the edge affects the function of placeholders.
Can we use the fact that it's a child of the edge to have the labels represent the connecting point by a name we can specify somehow in the data of the object?

**Reply by fermil** [reply]
This seems the latest discussion of this topic and it appears to be closed. Unfortuntaley on v.29.7.8 (the latest at the time I'm wiriting) the problem is still there. Either there is a genuine bug that hasn't been yet solved or instructuin on how to us this functionality are very poor (see attached screenshot)

<img width="498" height="440" alt="Example" src="https://github.com/user-attachments/assets/87cb8602-5fe3-4ccc-94da-849e44291076" />

**Reply by alderg** [reply]
Please share the .drawio file.

**Reply by fermil** [reply]
It's a very simple one
[Placeholders_ex.drawio](https://github.com/user-attachments/files/26891841/Placeholders_ex.drawio)

**Comment by alderg** [comment]
The label isn't an edge label. Placeholders are only resolved against the cell or any parent cell. Select the edge, press enter and add the label there, or double click on the edge to add an additional edge label (and enable placeholders on it).
